src/Components/Card.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import { Character } from "../API/useCharacters";
+import { useThemeContext } from "../Context/Theme-context";
+
+interface CardProps {
+  page: string;
+  results: Character[];
+}
+const CardContainer = styled(Link)<{ themeIsLight: boolean }>`
```

Comment:
In general we put the styled components in a dedicated file, located near the component file, with a similar name, like "[name].styles.ts". No need to change.

Reply:
It is good to know, I will apply in the future :).